data-discovery/use-cases.md

```diff
@@ -1,3 +1,5 @@
 # Use Cases for Data Discovery
 
 The following use cases must be considered when designing a mechanism for a data pod to define how its data is stored and indexed so that a given application can automatically find the data that it needs.
+
+ - Alice has a super file with a wide array of data that falls under many healthcare-related shapes on her Pod. She wants to share only data about ultrasound readings with her new OB-GYN without sharing information on her skin care, dentistry or other subjects. (Read-time shape validation)
```

Comment:
I don't see reason why use case should assume 'a super file'. Preferably it will just state 'a wide array of data' and leave it open to solution where data needs to stay in separate resources to manage access to them separately.